web/src/views/SubmissionPortal/Components/DataTypes.vue

```diff
@@ -42,9 +80,10 @@ export default defineComponent({
       value="mg"
       :disabled="templateChoiceDisabled"
       hide-details
+      @change="handleMetagenomeChange"
     />
     <div
-      v-if="multiOmicsForm.omicsProcessingTypes.includes('mg')"
+      v-if="showDataCompatibilityQuestions && multiOmicsForm.omicsProcessingTypes.includes('mg')"
```

Comment:
Would it make sense to check `multiOmicsForm.facilityGenerated === false` here and line 170 instead of creating a new prop? It seems as though that is the condition for these extra fields to show.

Reply:
No `multiOmicsForm.facilityGenerated === false` applies to both places where the `<DataTypes>` component is shown: https://github.com/microbiomedata/nmdc-server/blob/b1f8b7df96aeea9fe03217a2a7d960db2bc736a7/web/src/views/SubmissionPortal/Components/MultiOmicsDataForm.vue#L314-L325

I guess it would be possible to replace the new prop with `multiOmicsForm.doe === false`. But since we already have this separate `<DataTypes>` component, I figured that the prop helps keep the component be more encapsulated and less knowledgeable about the structure of `multiOmicsForm`.

Reply:
I think I might be confused. I thought that the first `<DataTypes>` was dependent on `multiOmicsForm.dataGenerated === false` and the second (the one that shows the doe compatibility questions) is dependent on `multiOmicsForm.facilityGenerated === false`

_data_ vs _facility_ generated.

Reply:
Whoops, my bad! I wasn't reading that closely enough 🤦🏻 

I think I still prefer passing in an explicit prop to `<DataTypes>`. You could make a similar argument about the existing `legend` prop. Could that text be changed from _within_ `<DataTypes>` by inspecting the `multiOmicsForm` state? Sure, but to me the explicitness of have a separate prop makes the behavior of the component more clear from the outside.

Reply:
Fair enough